courses/models_test.py

```diff
@@ -91,12 +92,34 @@ def test_program_page_properties():
     program = ProgramFactory.create()
     assert program.description is None
     assert program.duration is None
-    assert program.outcomes is None
-    program_page = ProgramPageFactory.create(
-        program=program, description="<p>desc</p>", duration="1 week"
+    ProgramPageFactory.create(
+        program=program,
+        title="<p>page title</p>",
+        subhead="subhead",
+        description="<p>desc</p>",
+        duration="1 week",
+        video_title="<p>title</p>",
+        video_url="http://test.com/mock.mp4",
+        background_image__title="background-image",
     )
+    assert program.display_title == "<p>page title</p>"
+    assert program.subhead == "subhead"
     assert program.description == "<p>desc</p>"
     assert program.duration == "1 week"
+    assert program.video_title == "<p>title</p>"
```

Comment:
why `html` is embedded in the fields ? Not every field allows html.

Reply:
This was part of #140 tests. That was approved as a rich text field for the video title.